resources/js/src/app/components/item/ItemImageCarousel.vue

```diff
@@ -1,35 +1,24 @@
 <template>
-    <div itemscope itemtype="http://schema.org/Thing">
-        <template>
-            <div class="single-carousel owl-carousel owl-theme owl-single-item mt-0" ref="single">
-                <div v-for="image in singleImages" class="prop-1-1">
-                    <a :href="image.url" :data-lightbox="'single-item-image' + _uid">
-                        <img class="owl-lazy" :data-src="image.url" :alt="getAltText(image)" :title="getImageName(image)">
-                    </a>
-                </div>
+    <div itemscope itemtype="'https://schema.org/Thing'">
+        <div ref="single" class="single-carousel owl-carousel owl-theme owl-single-item mt-0">
+            <div v-for="(image, index) in singleImages" :key="index" class="prop-1-1">
+                <a :href="webpImagesEnabled ? `${image.url}.webp` : image.url" :data-lightbox="`single-item-image${_uid}`">
+                    <lazy-img :alt="getAltText(image)" :image-url="`${image.url}.webp`" :fallback-url="image.url" :title="getImageName(image)" />
```

Comment:
Do we no longer need the `owl-lazy` class here?

Reply:
when keeping that class, it adds an ```opacity: 0``` to the image, hence the removal